mixer/test/client/istio_authn_peer_jwt_bound_peer/authn_report_test.go

```diff
@@ -36,24 +38,15 @@ const authnConfig = `
         }
       ],
       "principal_binding": 0
-    },
-    "jwt_output_payload_locations": {
-      "issuer@foo.com": "sec-istio-auth-jwt-output"
     }
   }
 `
 
 const secIstioAuthUserInfoHeaderKey = "sec-istio-auth-jwt-output"
 
-const secIstioAuthUserinfoHeaderValue = `
-{
-  "iss": "issuer@foo.com",
-  "sub": "sub@foo.com",
-  "aud": "aud1",
-  "non-string-will-be-ignored": 1512754205,
-  "some-other-string-claims": "some-claims-kept"
-}
-`
+const secIstioAuthUserinfoHeaderValue = `{"aud":"aud1","exp":20000000000,` +
```

Comment:
can you also add // nolint for this field.  the old format is easier to read

Reply:
I removed some unused variables but the nolint can not be added to that variable because it is used to compare with raw-claim, which does not have extra space characters.